src/modules/navigator/mission_block.cpp

```diff
@@ -160,6 +159,19 @@ MissionBlock::is_mission_item_reached_or_completed()
 			return false;
 		}
 
+	case NAV_CMD_DO_GIMBAL_MANAGER_PITCHYAW: {
+			const float gimbal_command_elasped_time_s = (now - _gimbal_command_time) * 1E-6f;
```

Comment:
```suggestion
			const float gimbal_command_elasped_time_s = (now - _gimbal_command_time) * 1E-6f;
```
You can use `hrt_elapsed_time()`. 
E.g. `if (hrt_elapsed_time(_gimbal_command_time) > _gimbal_wait_time * 1_s)`

Reply:
I used that form to be consistent with the `NAV_CMD_DO_WINCH` and `NAV_CMD_DO_GRIPPER` cases: https://github.com/PX4/PX4-Autopilot/blob/385fa260602476a1700d6a69ccd4c287d0558276/src/modules/navigator/mission_block.cpp#L176
Should I correct them all?

Reply:
I changed all of them.

Reply:
Thanks!